site/content/docs/5.2/helpers/stacks.md

```diff
@@ -22,9 +22,9 @@ Use `.vstack` to create vertical layouts. Stacked items are full-width by defaul
 <!-- Boosted mod: light border -->
 {{< example >}}
 <div class="vstack gap-3">
-  <div class="bg-light border border-light">First item</div>
-  <div class="bg-light border border-light">Second item</div>
-  <div class="bg-light border border-light">Third item</div>
+  <div class="bg-body-tertiary border border-light">First item</div>
+  <div class="bg-body-tertiary border border-light">Second item</div>
+  <div class="bg-body-tertiary border border-light">Third item</div>
```

Comment:
**[Bootstrap Side]** Couldn't we use something similar to what's done with flex and grid ?

Reply:
See https://github.com/twbs/bootstrap/pull/37900.